crowbar_framework/app/models/service_object.rb

```diff
@@ -758,7 +774,7 @@ def _proposal_update(bc_name, inst, proposal, validate_after_save = true)
       [200, {}]
     rescue Net::HTTPServerException => e
       Rails.logger.error ([e.message] + e.backtrace).join("\n")
-      [e.response.code, {}]
+      [e.response.code, e.message]
```

Comment:
That would be a chef-related exception, I'm not sure we want to expose the message to the user.